src/auth/executableresponse.ts

```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 Google LLC
```

Comment:
nit: I would call this `executable-reponse.ts`, there's a chance some of our files don't follow this convention, but it tends to be how files are named in JS.

Reply:
is there something special about this file that means it should have the hyphen or should all the new files have that format?

Reply:
Ideally we'd have a hypten separating words:

`executable-response.ts`, `external-client.ts`, etc., just a stylistic nit we could fix later -- I realize the codebase isn't consistent.